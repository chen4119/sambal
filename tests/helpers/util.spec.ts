import {
    frontMatter
} from "../../src/helpers/util";


describe("util", () => {

    describe("frontMatter", () => {
        const md = `
            ---
            test: hello world
            ---
            body of the markdown
            
            \`\`\`md
                ---
                test: hello world
                ---
                something
            \`\`\`
        `;
        it('return original id', async () => {
            const result = frontMatter(md);
            expect(result).toMatchSnapshot();
        });
    
    });

});