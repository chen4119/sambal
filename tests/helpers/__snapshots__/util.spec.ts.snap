// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util frontMatter return original id 1`] = `
Object {
  "content": "
            body of the markdown
            
            \`\`\`md
                ---
                test: hello world
                ---
                something
            \`\`\`
        ",
  "frontMatter": "
            test: hello world
            ",
}
`;
